.github/styles/vocab.txt

```diff
@@ -149,6 +149,7 @@ Mkdocs
 monorepo
 Monorepo
 monorepos
+msgraph
```

Comment:
I have to add it here, because the API reports (markdown files) contain the package name in the header. Would be cool if the package name is escaped as code, but that isn't something I can fix in Backstage.